_i18n/pt.yml

```diff
@@ -0,0 +1,63 @@
+# 1.0.0
+langs:
+  pt: Português
+
+global:
+  404: Página não encontrada
+home:
+  title: |
+    <span class="red">Força do trabalhador</span> na indústria de tecnologia
+
+  summary: Guiados por nossa visão de uma indústria de tecnologia inclusiva e equitativa, a TWC se organiza para aumentar o poder do trabalhador por meio da auto-organização e educação.
+  black_lives_matter:
+    title: Vidas Negras Importam
+    description:
+      Encorajamos que trabalhadores da indústria de tecnologia se posicionem ao lado do movimento Vidas Negras Importam na luta por justiça. Para isso, nós reunimos <a href="/security">uma lista de guias de segurança aqui</a> (em inglês) que você pode usar para ajudar a proteger a si mesmo e outres camaradas.
+  who_we_are:
+    title: Quem somos
+    description: Somos uma aliança de trabalhadores dentro e ao redor da indústria de tecnologia, organizadores sindicais, organizadores de comunidades e amigos.
+  why_we_organize:
+    title: Por que nos organizamos
+    description: Organizamos atividades de ativismo, engajamento cívico e educação nos Estados Unidos, India, Inglaterra, Alemanha e Brasil.
+  who_we_support:
+    title: Quem apoiamos
+    description: Trabalhamos em solidariedade com os movimentos existentes em direção à justiça social, direitos dos trabalhadores e inclusão econômica.
+  how_we_work:
+    title: Como trabalhamos
+    description: |
+      Somos uma organização democraticamente estruturada, totalmente voluntária e liderada por trabalhadores. Nesse momento, a associação consiste em participar de reuniões pessoalmente e trabalhar em vários projetos nos quais as pessoas estão interessadas. Organizamos on-line, mas os encontros em vida real são o ponto crucial do que fazemos. Confira nosso <a href="/community-guide">Guia da comunidade</a>  para mais informações. Também entre no <a href="/subscribe">Slack</a> para se conectar com pessoas em sua região.
+
+  check_out_meeting:
+    title: Confira uma reunião
+    locations: |
+      Atualmente, temos reuniões regulares em
+      <a href="/austin">Austin</a>,
+      <a href="https://twitter.com/twc_bangalore">Bangalore</a>,
```

Comment:
`<a href="/brasil">Brasil</a>,` should be added here and all other languages, ideally at least in English too, because that's the landing page. 

As a separate refactoring exercise, we could try to use the config chapter_collections, but right now it's just duplicated, because of the translation issue

Reply:
done